src/ResourceManagement/OperationalInsights/Microsoft.Azure.Management.OperationalInsights/Generated/Models/Tag.cs

```diff
@@ -0,0 +1,60 @@
+// Code generated by Microsoft (R) AutoRest Code Generator 1.0.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.Management.OperationalInsights.Models
+{
+    using System.Linq;
+
+    /// <summary>
+    /// A tag of a saved search.
+    /// </summary>
+    public partial class Tag
```

Comment:
This tag is not the tags as in a Resource. This tag is a special concept specific to our search.